README.md

```diff
@@ -2,79 +2,95 @@
 
 [![Build Status](https://travis-ci.org/alphagov/verify-service-provider.svg?branch=master)](https://travis-ci.org/alphagov/verify-service-provider)
 
-GOV.UK Verify uses SAML (Security Assertion Markup Language) to securely exchange information about identities. A Relying Party can use Verify Service Provider to generate and translate SAML communication to and from the Verify Hub.
+The Verify Service Provider (VSP) generates and translates Security Assertion Markup Language (SAML) messages to and from the GOV.UK Verify Hub. SAML is an open standard for secure message exchange which GOV.UK Verify uses when handling information about identities.
 
-Using Verify Service Provider will make it easier to:
-* connect multiple services with GOV.UK Verify - you only need one instance of Verify Service Provider
-* handle certificate rotations - you can host multiple certificates at a time
+Using the VSP removes the need for services to handle SAML by:
 
-You will need to host Verify Service Provider on your own infrastructure.
+* generating SAML requests to send to the GOV.UK Verify Hub
+* translating SAML responses from the GOV.UK Verify Hub into JSON
 
-Using Verify Service Provider is just one part of connecting to GOV.UK Verify. Refer to the [technical onboarding guide](https://www.docs.verify.service.gov.uk) for more information about connecting to GOV.UK Verify.
+Services will need to host the VSP on their own infrastructure.
+
+The VSP allows connected services to handle signing and encryption key rotation without service downtime.
```

Comment:
This feels a little out of place now. Maybe we could have it inside a subheading?